src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift

```diff
@@ -35,7 +35,6 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
 		setupBackButton()
 		
 		footerView?.primaryButton.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmissionHotline.primaryButton
-		footerView?.secondaryButton.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmissionHotline.secondaryButton
```

Comment:
Can we remove the `AccessibilityIdentifiers.ExposureSubmissionHotline.secondaryButton` also from the strings-file?

Reply:
No we need this